apps/web/app/components/FileUploadInput/ImageUploadInput.tsx

```diff
@@ -57,7 +57,7 @@ const ImageUploadInput = ({
         <input
           ref={fileInputRef}
           type="file"
-          accept=".png, .jpg, .jpeg"
+          accept=".png, .jpg, .jpeg, .svg"
```

Comment:
maybe we can create a variable from accepted?

Reply:
I think it should be stored in one place as a variable with descriptive name